pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -355,7 +355,9 @@ func (nm *NodeManager) DiscoverNode(nodeID string, searchBy cm.FindVM) error {
 				break
 			}
 		}
+	}
```

Comment:
Can we add a unit test for this multiple NIC case in nodemanager_test.go please?

Reply:
sure. sorry for being late.